### PR TITLE
[SLAS-393] Update all node-client method call to use SlashauthResponse type

### PR DESCRIPTION
**Pull request by donbernar**
## Refs

- **Issue**:  resolves [SLAS-393](https://getdebrief.atlassian.net/browse/SLAS-393)

## What?

All client methods in node-client now use SlashauthResponse type (example below), we have to update all reference to them.
```
const [value, meta, err] = await slashauthClient.[method]()
```

## Why?

To make extra fields such as response statusCode or error more accessible.

## Warnings (or deploy instructions)

- Pending update of version
- Pending metadata for users

### CODE REVIEW

File: src/controllers/user.ts [hunk 1]
```diff
@@ -87,6 +87,7 @@ export class UserController {
    * @param cursor
    * @returns
    */
+  // TODO ::::::::
```
Comment (donbernar) [inline src/controllers/user.ts:90]: Include metadata such as `cursor` and `hasMore` in the returning metadata object.

File: package.json [hunk 2]
```diff
@@ -21,8 +21,9 @@
     "@aws-sdk/client-secrets-manager": "^3.131.0",
     "@aws-sdk/lib-dynamodb": "^3.131.0",
     "@slashauth/express": "^0.2.3",
-    "@slashauth/node-client": "^0.9.2",
-    "@slashauth/types": "^0.3.1",
+    "@slashauth/node-client": "file:../slashauth-node-client/packages/core",
+    "@slashauth/packages": "file:../slashauth-node-client",
+    "@slashauth/types": "file:../slashauth-node-client/packages/types",
```
[the remark this answers is not in the document]
Comment (nedrocks) [reply package.json:26]: What is the PR for the node client? When will it be published?

File: src/controllers/app.ts [hunk 3]
```diff
@@ -13,23 +13,23 @@ export class AppController {
    * @returns
    */
   getAppMetadataController = async (clientID: string): Promise<AppRecord> => {
-    try {
-      const appResp = await slashauthClient.app.getInfo();
+      const [data, _, err] = await slashauthClient.app.getInfo();
```
Comment (nedrocks) [inline src/controllers/app.ts:16]: Ditto on either using `_` everywhere or making it unassigned.

**Review by nedrocks — APPROVED** [review]

**Comment by donbernar** [comment]
@nedrocks https://github.com/slashauth/slashauth-node-client/pull/26 <- This is the PR. Testing locally! Ill publish a version once I make sure everything works properly

**Pull request merged**